main.js

```diff
@@ -10,3 +12,142 @@ function toggle() {
+                      </div>
+                    `;
+  grid.appendChild(div);
+}
+
+// Popup-window cards
+
+for (let i = 0; i < obj.length; i += 1) {
+  const xxx = `.popping${i}`;
+  const open = document.querySelector(xxx);
+  open.addEventListener('click', () => {
+    const main = document.createElement('div');
+    main.className = 'main';
+    const popup = document.createElement('div');
+    popup.className = 'popup';
+    popup.innerHTML = `
+                            <div class="work-grid2">
+                                <div class="container3 spacing1">
+                                    <div class="item2-pop flex-column">
+                                        <button class="btn close" type="button"></button>
+                                        <img class="snapshoot2" src="${obj[i].image}" alt="portfolio project image ${i + 1}">
+                                    </div>
+                                    <div class="item-pop block-column">
+                                        <div class="wp-div">
+                                            <button class="button-interact antispacing1 display1"><a class="deco" href="${obj[i].live}" target="_blank">See Live <i
+                                                        class="i-back1"></i></a></button>
+                                            <button class="button-interact antispacing1 display1"><a class="deco" href="${obj[i].source}" target="_blank">See Source <i
+                                                        class="i-back2"></i></a></button>
+                                        </div>
+                                        <h3 class="h3">${obj[i].name}</h3>
+                                        <ul class="skills-wp">
+                                            <li class="skills1">${obj[i].technologies[0]}</li>
+                                            <li class="skills1">${obj[i].technologies[1]}</li>
+                                            <li class="skills1">${obj[i].technologies[2]}</li>
+                                            <li class="skills1">${obj[i].technologies[3]}</li>
+                                        </ul>
+                                        <p class="p3">${obj[i].description}</p>
+                                        <button class="button-interact antispacing1 display2"><a class="deco" href="${obj[i].live}" target="_blank">See Live <i
+                                                    class="i-back1"></i></a></button>
+                                        <button class="button-interact antispacing1 display2"><a class="deco" href="${obj[i].source}" target="_blank">See Source <i
+                                                    class="i-back2"></i></a></button>
+                                    </div>
+                                </div>
+                            </div>
+                    `;
+    main.appendChild(popup);
+    workSection.appendChild(main);
```

Comment:
- [ ] For the mobile version: the content of the popup isn't visible unless I zoom out, it's better if you make the popup scrollable so we can see all of the content 👍 

[ Optional ]
- [ ] To make the UX better, it's better if the body isn't scrollable when the modal is open.

## Your design:
![image](https://user-images.githubusercontent.com/50721479/202765320-6c29eb5d-17f4-4902-bc3e-1f23823dc090.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/202765359-9f4b3a3b-7c4c-4e01-8a6a-7a9c9872ca18.png)

Reply:
- [ ]  @HunterX-7 great work implementing the changes requested, only a small portion remaining, in the comparison below you can see the live demo and the GitHub button different in placement, you are aligned to the right, while in the design they occupy the left and right side, kindly check it;


Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/202841008-c57bd7b8-3f72-491f-bed4-fa42fc9352be.png) | ![image](https://user-images.githubusercontent.com/61408860/202841023-a0d954a5-812d-41b4-9887-e3817ccd321e.png)